src/ClockManager.cpp

```diff
@@ -1,36 +0,0 @@
-// Taken from Cornell's PAN
-
-#include "ClockManager.hpp"
-
-ClockManager::ClockManager(const unsigned int _control_cycle_size)
-    : TimedControlTask<void>(0), control_cycle_size(_control_cycle_size)
-{
-    initial_start_cycling_time = get_system_time();
-}
-
-void ClockManager::execute()
-{
-    unsigned int cycle_dt = 0;
-    if (has_executed) {
-        sys_time_t earliest_start_time = TimedControlTaskBase::control_cycle_start_time + control_cycle_size;
-        cycle_dt = wait_until_time(earliest_start_time);
-    }
-    has_executed = true;
-    TimedControlTaskBase::control_cycle_start_time = get_system_time();
-    control_cycle_count++;
-    sfr::mission::cycle_no++;
-    sfr::mission::boot_time_mins = sfr::mission::cycle_no % (constants::time::one_minute / constants::timecontrol::control_cycle_time_ms);
-    sfr::mission::cycle_dur = constants::timecontrol::control_cycle_time - cycle_dt;
```

Comment:
We should continue to track these three values in main control loop - personally I like having this stuff sill in clock manager, we could leave clock manager while still remove timed control tasks